### DISCUSSION TITLE
Paladin dropped Grief

### DISCUSSION THREAD

**Discussion by oden42** [isid0re/SoloLeveling#175] [Q&A]
I leveled up a new Smiter paladin and equipped him with a few items once he got to 80 to give him a boost. I made a phase blade grief and equipped him with it, and all was well until today he just auto-equipped a blue war scepter and trashed the grief.

Ever since then hes run out of cash and keeps dying, so I put his spirit sword back on until he can build himself a new grief.
He has also gone back to using blessed hammer + concentration aura instead of smite + fanaticism

Any idea why it may have done this? I updated the SoloLeveling files a day or two before this happened.

I love the scripts/bot! Thanks :)

**Comment by isid0re** [comment]
Usually with gear, the common mistake is pausing the bot adding items then unpausing the bot. This situation doesnt let the bot recognize the gear when it's in town. So it effectively is viewed as junk And discarded. 

If you want to gear the bot the best path is to put the items in the stash and then stop the bot then restart it. Also, it's best to make sure you have a pickit in place to keep those items just incase. So it doesnt try and drop or sell them.

**Reply by oden42** [reply]
Any idea why it is stuck using blessed hammer + concentration aura instead of smite + fanaticism? The bot respecced to max smite but ever since it dropped Grief it will only use blessed hammer.

Thanks
